database/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.8'
```

Comment:
Shouldn't we have a single docker-compose file for the entire project? The modules should be managed by separate `Dockerfile`s

Reply:
Sound's like a good idea to have `Dockerfile`s and top level `docker-compose.yaml`, but i'd keep that `database/docker-compose.yaml`. We can later incude that in top level `docker-compose.yaml` as described [here](https://docs.docker.com/compose/compose-file/14-include/)

Reply:
Okay, if there's a possibility to have docker-compose files including each other, I'm fine with that.